opencti-platform/opencti-front/src/private/components/data/DataTableToolBar.jsx

```diff
@@ -1910,6 +1925,36 @@ class DataTableToolBar extends Component {
                   </IconButton>
                 </div>
                 <div>
+                  {markAsReadEnabled && (
+                    <>
+                      <Tooltip title={t('Mark as read')}>
+                        <span>
+                          <IconButton
+                            aria-label="ack"
```

Comment:
```suggestion
                            aria-label={t('Mark as read')}
```
Aria label are for accessibility, like screen readers.
Basically, it should be equal to the button label.